README.md

```diff
@@ -227,7 +225,7 @@ having to manually rebuild each time.
 
 First clone and build `matrix-js-sdk`:
 
-1. `git clone git@github.com:matrix-org/matrix-js-sdk.git`
+1. `git clone https://github.com/matrix-org/matrix-js-sdk.git`
```

Comment:
TBH I think those who prefer HTTPS can feel free to change the URLs and they'd likely know how.

Reply:
it should be consistent in the readme, at the top you used https, so I changed it all to https. I personally prefer the https-urls in Readmes, because you can copy paste them to open the github page directly in the browser, but if you prefer git@... urls, then the line in the https://github.com/vector-im/riot-web/blob/master/README.md#building-from-source block shoud be changed instead

Reply:
>it should be consistent in the readme

This is fair enough. 

>you can copy paste them to open the github page directly in the browser

Also fair. Okay, this WFM and we can apply my previous comment to those who prefer TLS.